### ISSUE TITLE
:s command errors without in pairs/ in ipairs.

### ISSUE THREAD

**Issue by Lethalitics** [Epix-Incorporated/Adonis#893]
- What are the steps that lead to the bug occurring? Or, how frequently does the bug occur?
if you run a script line that only does "for i, v in Player:GetChildren()" for example, then the command will error.

- What would be the normally expected or desired result?
The command should run as intended.

- What is the actual result (the bug)?
Command Errors.

- What is the impact of this bug?
Less knowledgable players may not be able to understand the bug/why their command works on roblox but not via the s command. Fortunately able to be fixed by just adding ipairs/pairs.

- (Any additional context or information eg. in the form of error messages or screenshots.)

**Comment by Expertcoderz** [comment]
Not really a bug per se, but just because Adonis code execution currently only supports Lua (which Roblox previously used) and thus Luau-exclusive syntax and features such as __iter, compound assignments, type annotations etc. won't be compatible at the moment.

**Comment by P3tray** [comment]
> Not really a bug per se, but just because Adonis code execution currently only supports Lua (which Roblox previously used) and thus Luau-exclusive syntax and features such as __iter, compound assignments, type annotations etc. won't be compatible at the moment.

In plain english, use ipairs(Player:GetChildren()).

**Comment by WaviestBalloon** [comment]
This is not really a bug, more of the fact that the sandbox module FiOne does not support loops without `pairs`, `ipairs`, etc...
